dwarf.c

```diff
@@ -1615,8 +1615,7 @@ unit_addrs_search (const void *vkey, const void *ventry)
 
 static int
 resolve_unit_addrs_overlap_walk (struct backtrace_state *state,
```

Comment:
I'm not sure if this code path is covered by existing tests. Should I add a unit test?